src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -398,6 +399,12 @@ private function getArgumentsAsPhp(\DOMElement $node, $name, $lowercase = true)
                 case 'expression':
                     $arguments[$key] = new Expression($arg->nodeValue);
                     break;
+                case 'closure-proxy':
+                    if (2 !== count($id = explode(':', $arg->getAttribute('id')))) {
```

Comment:
would be better to use ``id`` and ``method`` IMO rather than exploding. It would be more consistent with other XML configuration settings